Resources/ServerInfo/Rules.txt

```diff
@@ -13,70 +13,67 @@
-
-[color=#7ade00]10.[/color] Do not disconnect, ghost, suicide, or otherwise intentionally permanently remove yourself from the round as important roles or while facing in-character punishment such as jail time without ahelping before hand. You don't need a response back, but failure to do so will result in consequences.
+[color=#7ade00]10.[/color] Do not disconnect, ghost, suicide, or otherwise intentionally permanently remove yourself from the round as an important role or while facing in-character punishment such as jail time without ahelping beforehand. You do not need to wait for a response, however, failure to notify admins will result in consequences.
 - This does not apply to cryo.
 
 
 [color=#7ade00]11.[/color] Your character name should be believable and realistic.
-- Your character name should not mimic popular real life or fictional people. Even if it's "clever" plays on them, like "White Walter."
-- Your character's name can't be phonetic word plays, like, "Mike Oxlong" or "Ben Dover."
-- Your name shouldn't be overly dumb or gamey either, like, "John Graytide."
-- Names should not include titles, such as, "Doctor", "Sergeant", or "Father."
+- Your character name should not mimic popular real-life or fictional people, even if it is a "clever" plays on them, such as "White Walter."
+- Your character's name should not be phonetic word plays, such as "Mike Oxlong" or "Ben Dover."
+- Your name should not be overly dumb or gamey either, such as "John Graytide."
+- Names should not include titles, such as "Doctor," "Sergeant," or "Father."
 
-[color=#7ade00]12.[/color] Do not prepare items or locations for "just in case" purposes. Referred to as powergaming. Standard Operating Procedure dictates how areas should be locked down and what gear should be held during any given circumstance. Minor things like getting a crowbar on round start is fine, seeking out weapons is not.
+[color=#7ade00]12.[/color] Do not prepare items or locations for "just in case" purposes (also referred to as "powergaming"). Standard Operating Procedure (SOP) dictates how areas should be locked down and what gear should be held and/or equipped during any given circumstance. Minor things such as getting a crowbar on round start is fine, seeking out weapons is not.
 
 [color=#ffe600]Roleplay Focused.[/color]
 
-[color=#7ade00]13.[/color] While this is HRP, avoid abandoning your job entirely for no good reason. You can focus on and off of work or ask the Head of Personnel to clock you out if you plan to be away for quite some time. Don't spend the entire round away from your job while on duty and your department needs you.
+[color=#7ade00]13.[/color] Because this is an HRP server, you should avoid abandoning your job for no good reason. You can focus on and off of work, or ask the Head of Personnel to clock you out if you plan to be away for quite some time. Don't spend the entire round away from your job while on duty and your department needs you.
```

Comment:
The "**While** this is an HRP server" was intentional, because people may see HRP as entirely roleplay and avoid doing their jobs.

Reply:
(That being said it's not a big deal)

Reply:
From an outside perspective, the sentence is unclear and difficult to understand.

Reply:
Ye, I can see that. Fine with that then.